### DISCUSSION TITLE
Different elevation for points in Coverage map

### DISCUSSION THREAD

**Discussion by argyrioskyrgiazos** [NVlabs/sionna#339] [Q&A] [answered]
hi,

I have been starting using Sionna RT, and I was curious whether it would be possible to compute a coverage map of a scene using different elevation for the points.

The objective of the above is somehow to take into account the terrain height. Now, as far as I know (There is another question about this), although the terrain elevation can be incorporated in a scene, the elevation of each simulated receiver is fixed. And I was wondering if it is possible a workaround to take the terrain heigh into account by tuning the receiver height in the coverage map.

Thank you in advance

**Comment by faycalaa** [comment, marked as answer]
Hi,

Since a coverage map is defined with respect to a measurement plane, all points for which the path loss is computed are considered to belong to the same flat surface. Note that the position, orientation, and size of the measurement plane can be set.

You could maybe compute multiple smaller coverage maps, each with different positions (elevations) and orientations, to approximate the shape of the ground.

Hope this helps.

**Reply by argyrioskyrgiazos** [reply]
Thank you for your reply!
